Sources/Private/CoreAnimation/Animations/CustomPathAnimation.swift

```diff
@@ -65,7 +65,7 @@ struct BezierPathKeyframe {
     -> KeyframeGroup<BezierPathKeyframe>
   {
     guard
-      let cornerRadius = cornerRadius,
```

Comment:
wheeee

Reply:
the linter automatically applied Swift 5.7 features after updating `Package.swift` to `swift-tools-version: 5.7`.